python/labbox_ephys/api/_workersession.py

```diff
@@ -153,20 +155,32 @@ def handle_message(self, msg):
             job = self._jobs_by_id[job_id]
             job.cancel()
     def iterate(self):
+        subfeed_watches = {}
         if (self._feed_uri is not None) and (self._feed_uri.startswith('feed://')):
-            subfeed_watches = {}
             for key in ['recordings', 'sortings']:
                 subfeed_name = dict(key=key, documentId=self._document_id)
                 subfeed_watches[key] = dict(
                     feedId=self._feed._feed_id, # fix this
                     subfeedName=subfeed_name,
                     position=self._subfeed_positions[key]
                 )
+        for w in self._additional_subfeed_watches:
```

Comment:
On each iteration we check for updates in the subfeeds that are being watched (incl curation subfeed)